README.md

```diff
@@ -1,2 +1,101 @@
-# biokotlin-tools
+
+# BioKotlin Tools
 Command line tools related to BioKotlin
+
+## Overview
+**BioKotlin Tools** is a command-line toolkit designed for genomic data processing and analysis. It includes a variety of utilities to handle and manipulate Variant Call Format (VCF) and Mutation Annotation Format (MAF) files. This toolkit is particularly useful for creating, validating, merging, and analyzing genomic variant data.
+
+## Installation
+To install BioKotlin Tools, download the package and make sure you have Java 17 or higher installed.
+
+## Usage
+The main entry point for BioKotlin Tools is the `biokotlin-tools` executable. The general usage pattern is:
+
+```
+biokotlin-tools [<options>] <command> [<args>]...
+```
+
+### Global Options
+- `--version`: Show the version of BioKotlin Tools and exit.
+- `-h, --help`: Show help message and exit.
+
+## Commands
+BioKotlin Tools offers several commands, each with specific options and arguments:
+
+### 1. maf-to-gvcf-converter
+Converts a MAF file to a GVCF file.
+
+**Usage:**
+```
+biokotlin-tools maf-to-gvcf-converter [<options>]
+```
+
+**Options:**
```

Comment:
It might be helpful to indicate which options are required - perhaps by adding (REQUIRED)/(OPTIONAL) or something similar on each line.  This comment applies to the options for all commands.

Reply:
I added (Required) to the required parameters